enterprise/cypher/acceptance-spec-suite/src/test/resources/cypher/features/DurationBetweenAcceptance.feature

```diff
@@ -359,25 +517,109 @@ Feature: DurationBetweenAcceptance
+      | 'PT8090H19M23.857S' |
+      | 'PT17544H4M45.999S' |
+      | 'PT8760H59M55.999S' |
+      | 'PT-5H-10M-36.143S' |
+      | 'PT-4H-10M-36.143S' |
+    And no side effects
+
+  Scenario: Should compute duration between if they differ only by a fraction of a second and the first comes after the second.
+    Given an empty graph
+    When executing query:
+      """
+      RETURN duration.seconds(localdatetime("2014-07-21T21:40:36.143"), localdatetime("2014-07-21T21:40:36.142")) as d
+      """
+    Then the result should be, in order:
+      | d |
+      | 'PT-0.001S' |
+    And no side effects
+
+  Scenario: Should handle durations at daylight saving time day
+    Given an empty graph
+    When executing query:
+    """
+    UNWIND[ duration.hours(datetime({year: 2017, month: 10, day: 29, hour: 0, timezone: 'Europe/Stockholm'}), localdatetime({year: 2017, month: 10, day: 29, hour: 4})),
+            duration.between(datetime({year: 2017, month: 10, day: 29, hour: 0, timezone: 'Europe/Stockholm'}), localdatetime({year: 2017, month: 10, day: 29, hour: 4})),
+            duration.hours(datetime({year: 2017, month: 10, day: 29, hour: 0, timezone: 'Europe/Stockholm'}), localtime({hour: 4})),
+            duration.between(datetime({year: 2017, month: 10, day: 29, hour: 0, timezone: 'Europe/Stockholm'}), localtime({hour: 4})),
+            duration.hours(localdatetime({year: 2017, month: 10, day: 29, hour: 0 }), datetime({year: 2017, month: 10, day: 29, hour: 4, timezone: 'Europe/Stockholm'})),
+            duration.between(localdatetime({year: 2017, month: 10, day: 29, hour: 0 }), datetime({year: 2017, month: 10, day: 29, hour: 4, timezone: 'Europe/Stockholm'})),
+            duration.hours(localtime({hour: 0 }), datetime({year: 2017, month: 10, day: 29, hour: 4, timezone: 'Europe/Stockholm'})),
+            duration.between(localtime({hour: 0 }), datetime({year: 2017, month: 10, day: 29, hour: 4, timezone: 'Europe/Stockholm'})),
+            duration.hours(date({year: 2017, month: 10, day: 29}), datetime({year: 2017, month: 10, day: 29, hour: 4, timezone: 'Europe/Stockholm'})),
+            duration.between(date({year: 2017, month: 10, day: 29}), datetime({year: 2017, month: 10, day: 29, hour: 4, timezone: 'Europe/Stockholm'})),
+            duration.hours(datetime({year: 2017, month: 10, day: 29, hour: 0, timezone: 'Europe/Stockholm'}), date({year: 2017, month: 10, day: 30}))
+          ] as d
+    RETURN d
+    """
+    Then the result should be, in order:
+      | d      |
+      | 'PT5H' |
+      | 'PT5H' |
+      | 'PT5H' |
+      | 'PT5H' |
+      | 'PT5H' |
+      | 'PT5H' |
+      | 'PT5H' |
+      | 'PT5H' |
+      | 'PT5H' |
+      | 'PT5H' |
+      | 'PT25H' |
+    And no side effects
+
+  Scenario: Should handle large durations
+    Given an empty graph
+    When executing query:
+    """
+    UNWIND[ duration.between(date("0001-01-01"), date("2001-01-01")),
+            duration.seconds(date("0001-01-01"), date("2001-01-01"))
```

Comment:
If you really want to look at big durations: How long ago did the dinosaurs die? When is the sun going to explode?